release/__init__.py

```diff
@@ -122,6 +122,16 @@ def load_providers():
             for name in provider_names}
 
 
+def validate_cloudformation(template_url):
```

Comment:
If at all possible I'd like to get this change out of the way / scope of the pulling apart bootstrap packages (otherwise it's just more stuff that all has to be ready for any of it to land)